helpers/getImage.js

```diff
@@ -19,8 +20,8 @@ const factory = globals => {
 
         if (_.isPlainObject(presets) && _.isPlainObject(presets[presetName])) {
             // If preset is one of the given presets in _images
-            width = parseInt(presets[presetName].width, 10) || 4096;
-            height = parseInt(presets[presetName].height, 10) || 4096;
+            width = parseInt(presets[presetName].width, 10) || 5120;
+            height = parseInt(presets[presetName].height, 10) || 5120;
```

Comment:
adjusted the maximums to match bcapp's while I was in here.

Reply:
Can we add a ticket to seed these from the `siteSettings` context so it always gets the latest value from bcapp.

Reply:
Will do.